### DISCUSSION TITLE
How do I get my Onderhorst cabinet organ sample set listed on the Wiki?

### DISCUSSION THREAD

**Discussion by BasKlb** [GrandOrgue/grandorgue#1656] [General]
Because KPN has stopped hosting homepages, I had to find a new host for my website where a free download of my sample set of the Onderhorst cabinet organ can be requested. The new address is: https://legro.free.nf.
I would like to see this sample set included in the list of free downloadable samples in the WIKI. How do I get that done?

Bas

**Comment by larspalo** [comment]
@BasKlb I've added it to the Wiki sample set list.

**Comment by BasKlb** [comment]
Thank you Lars,

Best regards, 

Bas
